docs/requirements.txt

```diff
@@ -5,5 +5,6 @@ sphinx-design
 sphinxemoji
 Jinja2>=3.0.0
 protobuf==3.20.1
+Pygments==2.11.2
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏Can we add a min version to avoid too tight of a dependency?

Reply:
I think the issue is coming with the most recent version (it goes 2.11.2 then 2.12.0) and people have reported 2.11.2 working. Maybe we could do a max version? I agree I'd rather not pin it so directly 😕